capabilities/discovery/tkg/capabilities.go

```diff
@@ -70,23 +66,23 @@ func clusterTypeFromMetadataConfigMap(ctx context.Context, c client.Client) (str
 		return "", err
 	}
 
-	data, ok := cm.Data["metadata.yaml"]
-	if !ok {
-		return "", fmt.Errorf("failed to get cluster type: metadata.yaml key not found in configmap %s", key.String())
-	}
+	//data, ok := cm.Data["metadata.yaml"]
```

Comment:
will undo these commented lines after `pkg/v1/tkg/test` package has been refactored and i believe @sathyanarays is working on it.

Reply:
I guess refactoring is still ongoing?